packages/extensions/src/yjs/index.ts

```diff
@@ -0,0 +1,83 @@
+import {
+  defineCommands,
+  defineKeymap,
+  definePlugin,
+  isApple,
+  Priority,
+  union,
+  withPriority,
+  type Keymap,
+} from '@prosekit/core'
+import type { Plugin } from '@prosekit/pm/state'
+import { yCursorPlugin, ySyncPlugin } from 'y-prosemirror'
+import { Awareness } from 'y-protocols/awareness'
+import * as Y from 'yjs'
+
+import { redo, undo, yUndoPlugin, type YUndoOpts } from './undo-plugin'
+
+export type YSyncOpts = Parameters<typeof ySyncPlugin>[1]
+
+function defineSyncPlugin(fragment: Y.XmlFragment, opts?: YSyncOpts) {
```

Comment:
Let's put `fragment` into `YSyncOpts` so that this function only has one paramter `options`.